src/components/Tooltip/TooltipCard.js

```diff
@@ -0,0 +1,67 @@
+import { formatAmount, USD_DECIMALS } from "../../lib/legacy";
+
+export default function TooltipCard({
+  title,
+  total,
+  avax,
+  arbitrum,
+  showDollar = true,
+  decimalsForConversion = USD_DECIMALS,
+  symbol,
+}) {
+  return (
+    <>
+      <p className="Tooltip-row">
+        <span className="label">{title} on Arbitrum:</span>
+        <span className="amount">
+          {showDollar && "$"}
+          {formatAmount(arbitrum, decimalsForConversion, 0, true)}
+          {!showDollar && symbol && " " + symbol}
+        </span>
+      </p>
+      <p className="Tooltip-row">
+        <span className="label">{title} on Avalanche:</span>
+        <span className="amount">
+          {showDollar && "$"}
+          {formatAmount(avax, decimalsForConversion, 0, true)}
+          {!showDollar && symbol && " " + symbol}
+        </span>
+      </p>
+      <div className="Tooltip-divider" />
+      <p className="Tooltip-row">
+        <span className="label">Total:</span>
+        <span className="amount">
+          {showDollar && "$"}
+          {formatAmount(total, decimalsForConversion, 0, true)}
+          {!showDollar && symbol && " " + symbol}
+        </span>
+      </p>
+    </>
+  );
+}
+
+export function TooltipCardRow({ label, value, showDollar = true, values }) {
```

Comment:
not sure, maybe better to use one `value` prop and check if it's an array?

2 separate props `value` and `values` might be confusing, what do you think?